src/Lykke.Logs/Obsolete/ILogEntityRowKeyGenerator.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+
+namespace Lykke.Logs
+{
+    /// <summary>
+    /// Log entry row key generator abstraction. Used by <see cref="AzureTableLogPersistenceQueue{TLogEntity}"/> to 
+    /// generate log entries row key 
+    /// </summary>
+    /// <typeparam name="TLogEntity">Log entry type</typeparam>
+    [Obsolete("Use new Lykke logging system")]
```

Comment:
Can we replace this to cref of a type from new logging system?

Reply:
There is no new type to replace this. This class is used in LWAPI to tace all requests. This will be replaced with general log entries with corresponding filtering